styles.css

```diff
@@ -147,13 +143,12 @@ margin: 4px 10px;
   padding: 4px;
   display: flex;
   flex-flow: row wrap;
-  justify-content: center;
-  align-items: center;
-  align-content: center;
+  justify-content: space-evenly;
 }
 
 .container-address > * {
   border: 1px solid none;
+  /* to check: How to get 'justify-content: space-between' between email and phone- otherwise delete */
```

Comment:
Did you find a solution, that fits for you well? - for me it seems you have chosen `space-evenly`now

Reply:
unfortunately not, I tried so many things and spend so much time with it but it is not working - i think i need to ask a coach later 😅